examples/guides/services/healthcare-service/tests/clemency-hospital-service-test.bal

```diff
@@ -0,0 +1,547 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+
+http:Client clientEPclemency = new("http://localhost:9090/clemency/categories");
+
+# Description: This test verifies if an appoinment can be reserved successfully. 
+# TC001 - Verify if appoinement reservation can be done by providing all the valid inputs. 
+# TC002 - Verify if appoinment reservation can be done by not providing non-mandatory feilds.
+# TC003 - Verify if appoinment reservation can be done for an unavailable doctor in the hospital. 
+#
+# + dataset - dataset Parameter Description
+@test:Config 
+{
+    dataProvider: "testReserveAppointmentDataProvider",
+     dependsOn: ["testAddDoctor"]
+}
+
+function testReserveAppointment(json dataset, json expectedStrings) 
+{
+    // set the json payload
+    http:Request request = new;
+    json payload = dataset;
```

Comment:
This is a redundant assignment. We can directly set 'dataset' as the request payload.

Reply:
Corrected.